### PR TITLE
Clear lower half of two-slot value when upper half is not live

### PR DESCRIPTION
**Pull request by zakkak**
Backport of: https://github.com/oracle/graal/commit/929dde28d8ba99b3d044be2b6fb86ce1625a2286
Fixes: #111

### CODE REVIEW

**Review by jerboaa — APPROVED** [review]
OK.

**Comment by zakkak** [comment]
Waiting for @Karm 's approval as well since this one affects 20.1

**Review by Karm — APPROVED** [review]
Thx

**Pull request merged**
